4266_Configs/vdmDriverII_Config_BCM1F_4266.json

```diff
@@ -94,7 +94,8 @@
   "Comment" : "Where to find the correction factors in json format",
   "InputDataDir" : "dataPrep_corr/corrData",
   "Comment": "This is relative to <AnalysisDir>",
-  "OutputSubDir": "corr"
+  "OutputSubDir": "corr",
+  "SingleBeamScans": [["X3", 2], ["Y3", 2], ["X4", 1], ["Y4", 1]]
```

Comment:
Put a comment line before line 98 explaining that the numbers correspond to beam 1/2 respectively